scrapy_statsd_extension/defaults.py

```diff
@@ -0,0 +1,6 @@
+STATSD_ENABLED = True
```

Comment:
Probably late asking about this, but what do the other built in middleware/extensions do for the default value for _ENABLED?

Reply:
I guess the default here should be False, most other plugins use it this way (crawlera / json rpc) and it makes more sense that way, it should be explicitly enabled rather than explicitly disabled